.gitlab/image_build/docker_windows.yml

```diff
@@ -48,7 +48,7 @@
   variables:
     IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent/agent
     BUILD_CONTEXT: Dockerfiles/agent
-    BUILD_ARG: --build-arg BASE_IMAGE=mcr.microsoft.com/powershell:nanoserver-${VARIANT} --build-arg WITH_JMX=${WITH_JMX}
+    BUILD_ARG: --build-arg BASE_IMAGE=mcr.microsoft.com/powershell:lts-windowsservercore-${VARIANT} --build-arg WITH_JMX=${WITH_JMX}
```

Comment:
Technically speaking we are moving back from `pwsh` 7.1 to 7.0. Probably not a big deal, but just to raise the point.

Reply:
I didn't catch that, thanks. It's probably fine, though, so I'll leave it like this.